src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -55,6 +55,16 @@ class TosService(val directoryDao: DirectoryDAO, val tosConfig: TermsOfServiceCo
       )
     )
 
+  def getTosText(docSet: Set[String]): IO[String] =
+    docSet match {
+      case set if set.isEmpty => IO.pure(termsOfServiceText)
+      case set if set.size == 1 && set.contains("privacyPolicy") => IO.pure(privacyPolicyText)
+      case set if set.size == 1 && set.contains("termsOfService") => IO.pure(termsOfServiceText)
+      case set if set.size == 2 && set.contains("privacyPolicy") && set.contains("termsOfService") =>
+        IO.pure(termsOfServiceText + "\n\n" + privacyPolicyText)
```

Comment:
Do we not want to send back a list of strings? That way, the caller can more easily format the data sent back?